packages/core/README.md

```diff
@@ -1,7 +1,29 @@
 # @contentful/experiences-core
 
-This package provides core internal functionality for the [Experiences SDK](https://www.contentful.com/developers/docs/experiences/set-up-experiences-sdk/).
+### Purpose
+- To contain shared code and utilities across the Experience-builder packages such as constants, types, transformers, and hooks.
+- This means additions to core should be framework agnostic. For example, code that is only compatible with React do not belong in the core package.
 
-## Documentation
+### Concepts
+- **Deep binding**: Relates to the ability to resolve n references. Such an entry containing a reference to another entry which may also contain a reference to another entry and so on.
+- **Component Definition** Baseline definition to outline what is needed to define all component types whether it is a structure, built-in, or custom component.
+- **Entity**: Contentful entries that get converted into an experience object that can be used within the experience builder packages.
+- **Fetchers**: The helper classes and functions in the Entity directory get consumed by the fetchers which get bundled up into either `fetchBySlug` or `fetchById`. The listed fetchers are the two main functions that are exposed to the end user in the sdk to fetch a user's experience which is stored in Contentful as an entry.
+- **Registries**: An in memory storage technique that the user can decide to provide for themselves or other personas in the editor canvas. Design Tokens is an useful example where a user can provide their own thematic values with regards to text size, component sizes, spacing, colors, and borders in a json format that is resolved by the registry and provided as options in the ui editor sidebar.
+- **Exporting types and constants**: Notice in the `package.json` there is a `typesVersions` definition that separates the constants and types as separated bundled packages where the rollup is defined in the `rollup.config.mjs` file. This effectively means in practice consuming the types and constants have different imports - namely `@contentful/experiences-core/constants` and `@contentful/experiences-core/types`.
+```
+  "typesVersions": {
+    "*": {
+      "types": [
+        "./dist/types.d.ts"
+      ],
+      "constants": [
+        "./dist/constants.d.ts"
+      ]
+    }
+  },
+```
 
-Please refer to our [Documentation](https://www.contentful.com/developers/docs/experiences/) to learn more about it.
+### Relevant Contentful Documentation Links
+[Data Structures](https://www.contentful.com/developers/docs/experiences/data-structures/)
+[Design Tokens](https://www.contentful.com/developers/docs/experiences/design-tokens/)
```

Comment:
Should this be a bulleted list so the links are on separate lines?  Currently it renders like this:
> ![Screenshot 2024-05-14 at 3 39 10 PM](https://github.com/contentful/experience-builder/assets/8539634/703e668a-5b76-4447-8c8e-b07c4794f15f)

Reply:
My plugin lied to me

![Screenshot 2024-05-15 at 9 43 20 AM](https://github.com/contentful/experience-builder/assets/124832189/cbf80e65-99fc-4c55-abec-4a7e8aba762e)

Reply:
Might be time to delete that extension! Vscode has native support for previewing markdown now and it renders very similar to github

![Screenshot 2024-05-15 at 10 32 48 AM](https://github.com/contentful/experience-builder/assets/8539634/6ac64138-f5be-49f8-ab74-3f97e2565bcb)


If you want make the two links appear on different line without using bullets, you can add a `\` to force a line break after the first link:

```md
### Relevant Contentful Documentation Links
[Data Structures](https://www.contentful.com/developers/docs/experiences/data-structures/)\
[Design Tokens](https://www.contentful.com/developers/docs/experiences/design-tokens/)
```